recipes/swmm-toolkit/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "swmm-toolkit" %}
+{% set version = "0.15.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/swmm-toolkit-{{ version }}.tar.gz
+  sha256: bc3d36e6cca656a6d18c4077b9fc429b54b65f8f7f9091dac5f5b7cd5a66f6dc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [py<38]
+  missing_dso_whitelist:  # [win]
+    - "*/swmm5.dll"  # [win]
+    - "*/swmm-output.dll"  # [win]
+  script_env:
+   - CFLAGS=-D_FORTIFY_SOURCE=2  # [osx]
+   - CXXFLAGS=-D_FORTIFY_SOURCE=2  # [osx]
```

Comment:
This overrides and not extends rhe flags, it should extend.

Reply:
Right, for now that's what we want, until we take care of the FORTIFY_SOURCE issue in the next release. Do you mean something like this in `script_env`: `CFLAGS="${CFLAGS} -D_FORTIFY_SOURCE=2"`

Reply:
Yes, exactly that.

Reply:
Done.

Reply:
The `osx` build throws this error: `export: '-D_FORTIFY_SOURCE=2': not a valid identifier`. Any ideas how to solve this? Maybe this cannot be done with `script_env`?

Reply:
@xhochy I fixed it by moving them to `script`.